test/Polly.Core.Tests/Timeout/TimeoutResilienceStrategyTests.cs

```diff
@@ -131,7 +132,12 @@ public async Task Execute_Timeout_EnsureStackTrace()
         ResilienceContextPool.Shared.Get(),
         "state");
         outcome.Exception.Should().BeOfType<TimeoutRejectedException>();
-        outcome.Exception!.StackTrace.Should().Contain("Execute_Timeout_EnsureStackTrace");
+
+#if NET8_0_OR_GREATER
+        outcome.Exception!.StackTrace.Should().Contain(nameof(Execute_Timeout_EnsureStackTrace));
+#else
+        outcome.Exception!.StackTrace.Should().Contain(nameof(StrategyHelper));
+#endif
```

Comment:
This fixes the tests, but I don't fully understand what's going on some I'm not sure this is the right thing to do.

Reply:
For some reason when .NET 8 is not involved the calling function `Execute_Timeout_EnsureStackTrace` is not included in the stack trace string.

Reply:
Happens only on ubuntu

Reply:
Is that a bug we should report, or is it an OS-specific implementation detail we just need to accept?

Reply:
Still investigating, however, to me it seems like a bug.